content/news/030/index.md

```diff
@@ -81,6 +81,11 @@ If needed, a section can be split into subsections with a "------" delimiter.
 <!-- Up to 10 links to interesting issues -->
 
 ## Other News
+Offline workshop in Ukraine. Write Game on Rust
```

Comment:
this section is supposed to be one-liners, but we definitely would want to mention this somewhere!

Reply:
Thanks for your responses @ozkriff , @AngelOnFira , @kvark ! 
Grateful for being open and friendly. You create a wonderful and useful newsletter and make a great contribution to the GameDev on Rust. It's impressive!

Reply:
@ozkriff  will this form of message be suitable for your  pages in [Twitter](https://twitter.com/rust_gamedev) and [Reddit](https://www.reddit.com/r/rust_gamedev)? 
… The Ukrainian Rust community holds an offline workshop “Write Game on Rust” in Kyiv. You can join the event on February 5-6 [here] (http://learntogether.pro/) …

Reply:
In which section of the newsletter should I insert the corrected information about the event?

Reply:
@VVPergat in case you haven't seen my telegram reply:

- I'm not really sure where it's better to place this announcement because, as discussed above, the event happens before the newsletter will be released. Maybe it's better to include it as a oneliner into this issue and add a separate section about the results into the next newsletter ([example](https://gamedev.rs/news/026#rust-graphics-meetup-1)). @17cupsofcoffee @AngelOnFira?
- We don't usually announce third-party events directly from the https://twitter.com/rust_gamedev account but rather retweet these announcements (if they mention rust_gamedev or include "#rustgamedev", "#rustlang #gamedev", etc tags).
- Here're examples of how rust graphics meetup was announced on Reddit: [1](https://reddit.com/r/rust_gamedev/comments/pjpzc9/rust_graphics_meetup_october_2nd_2021/), [2](https://reddit.com/r/rust_gamedev/comments/pwvq74/the_first_rust_graphics_meetup_is_happening_this/).